rasa/nlu/featurizers/mitie_featurizer.py

```diff
@@ -25,29 +35,42 @@ def ndim(self, feature_extractor: "mitie.total_word_feature_extractor"):
 
         return feature_extractor.num_dimensions
 
+    def get_tokens_by_attribute(self, example, attribute):
+
+        return example.get(MESSAGE_TOKENS_NAMES[attribute])
+
     def train(
         self, training_data: TrainingData, config: RasaNLUModelConfig, **kwargs: Any
     ) -> None:
 
         mitie_feature_extractor = self._mitie_feature_extractor(**kwargs)
         for example in training_data.intent_examples:
-            features = self.features_for_tokens(
-                example.get("tokens"), mitie_feature_extractor
-            )
-            example.set(
-                "text_features",
-                self._combine_with_existing_text_features(example, features),
-            )
+
+            for attribute in MESSAGE_ATTRIBUTES:
```

Comment:
I must say that this whole constants handling is really hard to read - not sure if there is a fix for this though